guide/10-mapping-and-visualization/usage-of-visualVariables-and-other-renderer-options-in-map-widget.ipynb

```diff
@@ -12,7 +12,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Here we need to distinguish between `autocast` and smart mapping. We can say that "While smart mapping will automatically choose the class breaks, users can use autocasting if they want finer control by providing their own class breaks and symbology for each class".</p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>